vars/runIntegTests.groovy

```diff
@@ -0,0 +1,14 @@
+void call(Map args = [:]) {
```

Comment:
I'm not sure this file as indirection is very useful the amount of code to call this function is nearly the same as this file.  Note; this would be more useful if it was automagically handling opensearch dashboards

FYI @tianleh

Reply:
Right, this script isn't doing so much now, and is really only abstracting out the job name. I'm ok to remove and add directly in distribution-build workflow.

Regarding dashboards, ideally the integ-test job won't need new parameters - [runIntegTestScript](https://github.com/opensearch-project/opensearch-build/blob/main/vars/runIntegTestScript.groovy) (which is called in the integ-test job) is where the logic can be added to handle dashboards. It may need some additional args to handle multiple build manifests, for example.

Reply:
Thanks for the discussion about the support for OpenSearch Dashboards. It will need the artifact url for Dashboards from its own distribution build job (similar to the one in this PR) and one additional url for OpenSearch (which will be the `latest` url). Maybe there will be some common logic for the first part. However, to avoid using if/else as branching logic, I could image that it is very possible to be two different files. I am ok to perform some refactoring once I start working on the Dashboards part (after your work is merged).